packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -2250,26 +1937,21 @@ class CupertinoDialogAction extends StatelessWidget {
     // button text to fit the available space.
     final double fontSizeRatio = MediaQuery.textScalerOf(context).scale(textStyle.fontSize!) / _kDialogMinButtonFontSize;
 
-    return IntrinsicHeight(
-      child: SizedBox(
-        width: double.infinity,
-        child: FittedBox(
-          fit: BoxFit.scaleDown,
-          child: ConstrainedBox(
-            constraints: BoxConstraints(
-              maxWidth: fontSizeRatio * (dialogWidth - (2 * padding)),
-            ),
-            child: Semantics(
-              button: true,
-              onTap: onPressed,
-              child: DefaultTextStyle(
-                style: textStyle,
-                textAlign: TextAlign.center,
-                overflow: TextOverflow.ellipsis,
-                maxLines: 1,
-                child: content,
-              ),
-            ),
+    return FittedBox(
```

Comment:
I don't know what the `IntrinsicHeight` and `SizedBox` were for, but removing them does not break anything even without the rest of the change in this PR.